lib/scan/detect_values.rb

```diff
@@ -23,6 +23,22 @@ def self.set_additional_default_values
       return config
     end
 
+    def self.filter_simulators(simulators, deployment_target)
+      # Filter out any simulators that are not the same major and minor version of our deployment target
+      if deployment_target.to_s.length > 0
```

Comment:
Comparing both major and minor versions. (so that "9.0" simulators don't match deployment_target's of "9.2", for instance).  Any suggestions for how to make this code more concise - it is definitely rather defensive.